docs/api/vast-client.md

```diff
@@ -99,6 +101,74 @@ Instance of a class which implements the `Storage` interface. Should be set up o
 
 ## Public Methods 💚 <a name="methods"></a>
 
+### addURLTemplateFilter(filter)
```

Comment:
Can you move thoses methods at the end ? We should keep the most important one at the top